mycore-base/src/main/resources/config/mycore.properties

```diff
@@ -35,6 +35,14 @@
   MCR.XMLParser.ValidateSchema=true
 # MCR.XMLParser.ValidateSchema=false
 
+##############################################################################
+# Servlet Filter configuration
+##############################################################################
+
+MCR.Filter.UserAgent.Pattern=(bot|spider|crawler|mercator|slurp|seek|nagios|Java|curl|wget|fetch)
```

Comment:
` .PatternInvalid` ?

Reply:
Maybe `MCR.Filter.UserAgent.Pattern.Bot` or more generally  `MCR.Filter.UserAgent.Pattern.Unacceptable`.

Something like `MCR.Filter.UserAgent.Pattern.Invalid` would be bad, because, according to class comment and, indeed, the code, matching agents are actually valid. They are just not welcome. Therefore, a match just results in a closed session, but the request is served anyways.

This confused me too, but the code basically does the following:
```
if (userAgent == null || isInvaild(userAgent)) {
   if (!ACCEPT_INVALID_USER_AGENTS) {
      return rejectRequest();
   }
}

if (isInvaild(userAgent) || isBot(userAgent)) {
   closeSession();
}

return acceptReuest();
```

Reply:
renamed to `MCR.Filter.UserAgent.BotPattern` in https://github.com/MyCoRe-Org/mycore/pull/1935/commits/bb9ca9654f7c2c16eb5e7a8189e5ed294ba214e5

Reply:
see https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#CASE_INSENSITIVE